### PR TITLE
Feature/36310 - Slider - dot support

### PR DESCRIPTION
**Pull request by kjugi**
+ Add dot support in slider
+ Update README.md
+ Add styles to slider

https://dev.snowdog.pro/issues/36310

### CODE REVIEW

File: components/03-modules/slider/slider.js [hunk 1]
```diff
@@ -6,24 +6,103 @@
...
+            lorySlider.slideTo(Array.prototype.indexOf.call(dots, e.target.parentNode));
+          }
+          else {
+            lorySlider.slideTo(Array.prototype.indexOf.call(dots, e.target));
+          }
+        });
+      })
+    }
+
+    function dotClassToggle(e) {
```
Comment (szafran89) [inline components/03-modules/slider/slider.js]: what is `e`? Is it `element` or `event`? Better use `el` id element.

File: components/03-modules/slider/_slider.scss [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+$slider__navigation-icon-background   : $white !default;
+$slider__icon-size                    : 25px !default;
+$slider__icon-fill                    : $color-primary !default;
+$slider__icon-fill-hover              : darken($color-primary, 10%) !default;
+$slider__dot-margin                   : $spacer--medium !default;
+$slider__dot-padding                  : $spacer--medium !default;
+$slider__dot-inside-size              : $spacer--medium !default;
+$slider__dot-inside-background        : $gray-dark !default;
+$slider__dot-inside-border-radius     : 8px !default;
+$slider__dot-inside-background\@active: $color-primary !default;
```
Comment (szafran89) [inline components/03-modules/slider/_slider.scss]: it is not break point.  Should we use here `@` not regular modifier?

File: components/03-modules/slider/_slider.scss [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
+$slider__navigation-icon-margin       : $spacer !default;
+$slider__navigation-icon-background   : $white !default;
+$slider__icon-size                    : 25px !default;
+$slider__icon-fill                    : $color-primary !default;
+$slider__icon-fill-hover              : darken($color-primary, 10%) !default;
+$slider__dot-margin                   : $spacer--medium !default;
+$slider__dot-padding                  : $spacer--medium !default;
+$slider__dot-inside-size              : $spacer--medium !default;
+$slider__dot-inside-background        : $gray-dark !default;
+$slider__dot-inside-border-radius     : 8px !default;
```
Comment (szafran89) [inline components/03-modules/slider/_slider.scss]: Better is to use `$spacer`

File: components/03-modules/slider/_slider.scss [hunk 4]
```diff
@@ -79,4 +80,37 @@ $slider__icon-fill-hover           : darken($color-primary, 10%) !default;
...
+    }
+
+    &__dot {
+        display: block;
+        padding: $slider__dot-padding;
+        margin-right: $slider__dot-margin;
+        cursor: pointer;
+
+        &--active {
+            & .slider__dot-inside {
```
Comment (szafran89) [inline components/03-modules/slider/_slider.scss]: useless space after `&`

File: components/03-modules/slider/slider.js (hunk 1 above)
Comment (kjugi) [reply components/03-modules/slider/slider.js]: renamed to event

File: components/03-modules/slider/_slider.scss (hunk 4 above)
Comment (kjugi) [reply components/03-modules/slider/_slider.scss]: not space can be useless b/c it's nested element so this space is required in this case. I can delete `&` sign to working fine too

File: components/03-modules/slider/slider.js (hunk 1 above)
Comment (kjugi) [reply components/03-modules/slider/slider.js]: other `e` sign too

File: components/03-modules/slider/_slider.scss (hunk 3 above)
Comment (kjugi) [reply components/03-modules/slider/_slider.scss]: true

File: components/03-modules/slider/_slider.scss (hunk 2 above)
Comment (kjugi) [reply components/03-modules/slider/_slider.scss]: replaced with regular modifier

File: components/03-modules/slider/_slider.scss (hunk 4 above)
Comment (szafran89) [reply components/03-modules/slider/_slider.scss]: ok so `&` is useless

Comment (szafran89) [reply components/03-modules/slider/_slider.scss]: Ok I see you fixed it.

**Pull request merged**